.changeset/stale-carpets-poke.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Externalize repository processing for BitbucketDiscoveryProcessor.
+
+Add an extension point where you can customize how a matched Bitbucket repository should
+be processed. This can for example be used if you want to generate the catalog-info.yaml
+automatically based on other files in a repository, while taking advantage of the
+build-in repository crawling functionality.
```

Comment:
Can you update the [Bitbucket Discovery doc](https://backstage.io/docs/integrations/bitbucket/discovery) with some instructions? A minor hint in the changeset might be nice too 😄  (e.g. "now accepts `options.parser` in the `fromConfig` method).

Reply:
Added some extra documentation to describe the functionality both in the changeset and the documentation.